src/ast/mod.rs

```diff
@@ -1534,1084 +1534,1086 @@ impl fmt::Display for Statement {
     // split up without extracting structs for each `Statement` variant.
     #[allow(clippy::cognitive_complexity)]
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        match self {
-            Statement::Kill { modifier, id } => {
-                write!(f, "KILL ")?;
-
-                if let Some(m) = modifier {
-                    write!(f, "{} ", m)?;
-                }
-
-                write!(f, "{}", id)
-            }
-            Statement::ExplainTable {
-                describe_alias,
-                table_name,
-            } => {
-                if *describe_alias {
-                    write!(f, "DESCRIBE ")?;
-                } else {
-                    write!(f, "EXPLAIN ")?;
-                }
+        stacker::maybe_grow(2 * 1024 * 1024, 5 * 1024 * 1024, || {
```

Comment:
I was worried about this indent change because it will conflict with every other PR. 

So I pushed 853a890 as a way to make this diff smaller and shrink the merge conflict problem